client/src/components/Tabs/TabNav.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Nav } from 'reactstrap';
+import PropTypes from 'prop-types';
+import Tab from './Tab';
+
+/**
+ * Builds the tabset navigation links, will hide the links if there is only one child
+ */
+function TabNav({ currentTab, children, onToggle }) {
+  const tabs = React.Children
+    .map(
+      children,
+      ({ props }) =>
+        (<Tab
+          {...props}
+          onToggle={() => currentTab !== props.name && onToggle(props.name)}
+          active={currentTab === props.name}
+        />)
+    );
+
+  return tabs ?
+    <Nav tabs role="tablist">{tabs}</Nav> :
+    null;
+}
+
+TabNav.propTypes = {
+  currentTab: PropTypes.string,
+  onToggle: PropTypes.func
```

Comment:
```suggestion
  onToggle: PropTypes.func
  name: PropTypes.name
```
Since using props.name in this file

Should probably make everything .isRequired too

Reply:
`name` is only use while looping over the children. It's not a prop of TabNav.